bindata/v4.1.0/kube-apiserver/pod.yaml

```diff
@@ -113,6 +115,39 @@ spec:
       requests:
         memory: 50Mi
         cpu: 10m
+  - name: kube-apiserver-watchdog-REVISION
+    command: ["cluster-kube-apiserver-operator", "file-watcher-watchdog"]
```

Comment:
you need `shareProcessNamespace: true` in spec (https://github.com/openshift/cluster-kube-controller-manager-operator/pull/254/files#diff-ad2ab26372ca50cd65844a937511d0a4R11)

Reply:
Yeah, I just noticed, thanks though :+1: